### PR TITLE
Create FakeMsgCmd

### PR DESCRIPTION
**Pull request by Nitrogen-Dioxide**
Generates a fake message into the chat screen making you able to fake
everything.

The image explains everything:

### CODE REVIEW

**Comment by labcraft59** [comment]
Great idea!

**Comment by Eyremba** [comment]
Great! Nice!

**Comment by Alexander01998** [comment]
I don't remember if there is an issue for this, but my plan was to change the `.wms echo` command so that it does this rather than adding a new command.

#1919 is about a different command (it would show a custom message on the "you were banned/kicked from the server" screen).

**Comment by PixelMaster3D** [comment]
Lets make it where you can have a different name from the server then.

**Comment by PixelMaster3D** [comment]
Then you can make another Pull for leave command. This was input on the wrong issue. Please ignore this Pull as .leave msg.

**Comment by KhantKyaw** [comment]
When will it add?

File: Wurst Client/src/tk/wurst_client/commands/CmdManager.java [hunk 1]
```diff
@@ -41,6 +41,7 @@ public CmdManager()
 		addCommand(new DamageCmd());
 		addCommand(new DropCmd());
 		addCommand(new EnchantCmd());
+		addCommand(new FakeMsgCmd());
```
Comment (KhantKyaw) [inline Wurst Client/src/tk/wurst_client/commands/CmdManager.java]: When will it add?

**Comment by PixelMaster3D** [comment]
This pull should be added and be linked to #1919.

**Pull request closed without merging**